wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -4,7 +4,7 @@
 
 {% block header %}
     {% page_permissions parent_page as parent_page_perms %}
-    {% include 'wagtailadmin/pages/page_listing_header.html' with title=header_title search_url=search_url page_perms=parent_page_perms %}
+    {% include 'wagtailadmin/pages/page_listing_header.html' with title=header_title search_url=index_results_url page_perms=parent_page_perms %}
```

Comment:
Following the code in the generic templates.

We should probably get rid of the generic `IndexView`'s `get_search_url` in the future